Makefile

```diff
@@ -119,7 +123,16 @@ integration: up ## Run the integration tests in a Docker container
 
 .PHONY: hbase-shell
 hbase-shell: ## Open an Hbase shell onto the running hbase container
-	docker-compose run --rm hbase shell
+	@{ \
+		export HBASE_TO_MONGO_EXPORT_VERSION=$(hbase_to_mongo_version); \
+		export AWS_DEFAULT_REGION=$(aws_default_region); \
+		export AWS_ACCESS_KEY_ID=$(aws_access_key_id); \
+		export AWS_SECRET_ACCESS_KEY=$(aws_secret_access_key); \
+		export S3_BUCKET=$(s3_bucket); \
+		export S3_PREFIX_FOLDER=$(s3_prefix_folder); \
+		export DATA_KEY_SERVICE_URL=$(data_key_service_url); \
+		docker-compose run --rm hbase shell; \
```

Comment:
I don't think you use any of these ENVs in the hbase image do you?

Reply:
no but they are in the compose file, you get misleading warnings otherwise